src/tools/sdk/_linux.md

```diff
@@ -9,28 +9,58 @@ versions are released.
 
 Perform the following **one-time setup**:
 
+{% if site.data.pkg-vers.SDK.channel == 'dev' %}
+```terminal
+$ sudo apt-get update
+$ sudo apt-get install apt-transport-https
+$ sudo sh -c 'curl https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -'
+$ sudo sh -c 'curl https://storage.googleapis.com/download.dartlang.org/linux/debian/dart_unstable.list > /etc/apt/sources.list.d/dart_unstable.list'
```

Comment:
To avoid duplication of the first few instructions, move `{% if site.data.pkg-vers.SDK.channel == 'dev' %}` to just above this line (refactor the else part accordingly).

Reply:
I tried that first, but the \`\`\` seemed to interfere with jekyll interpretation.

Reply:
Strange. Ok.